src/aasthaJJun20/FindIndexOfChars.java

```diff
@@ -0,0 +1,23 @@
+package aasthaJJun20;
+
+public class FindIndexOfChars {
+	void countChars(String input){
+		for(int index = 0; index < input.length(); index++){
+			int count = 0;
+			//char ch = input.charAt(index);
+			if(index == input.indexOf(input.charAt(index))){
+				for(int innerIndex = 0; innerIndex < input.length(); innerIndex++){
```

Comment:
Dear, here two for loop is not required. One is sufficient. Please optimize your code.

Reply:
Changes done as per you comment. please review. :)